code/modules/hydroponics/hydroponics.dm

```diff
@@ -784,6 +785,8 @@
 			T.myseed.set_potency(round((T.myseed.potency+(1/10)*(myseed.potency-T.myseed.potency))))
 			T.myseed.set_instability(round((T.myseed.instability+(1/10)*(myseed.instability-T.myseed.instability))))
 			T.myseed.set_yield(round((T.myseed.yield+(1/2)*(myseed.yield-T.myseed.yield))))
+			any_adjacent = TRUE
+			particles = new /particles/pollen()
```

Comment:
wouldn't this look weird or at least cause some visual dissonance if we keep regenerating particles when we don't need to (i.e. we remain adjacent)... might be a good idea to do an `(!isnull(particles))` check so we don't have to keep doing needless work IMO

Reply:
So true